Make missing known hosts file message a verbose message

tyll opened the pull request:
Since ansible does not completely implement a OpenSSH option parser it
does not really know which known hosts file needs to be used. Therefore
a faulty message appears if eg a GlobalKnownHosts file is used or a
non-standard UserKnownHosts file as specified by ~/.ssh/config or eg
/etc/ssh/ssh_config. Therefore the message a about the known hosts files
not being found should be only shown if VERBOSITY is at least 1.

mpdehaan commented:
I disagree with the above assertion that we should hide the message unless verbosity is set, and think it would be more interesting to get at the root of the underlying problem.

Further, this patch breaks attribution of numerous lines by cleaning up trailing whitespace, so it should not do this.

tyll committed:
Make missing known hosts file message a verbose message

Since ansible does not completely implement a OpenSSH option parser it
does not really know which known hosts file needs to be used. Therefore
a faulty message appears if eg a GlobalKnownHosts file is used or a
non-standard UserKnownHosts file as specified by ~/.ssh/config or eg
/etc/ssh/ssh_config. Therefore the message a about the known hosts files
not being found should be only shown if VERBOSITY is at least 1.

tyll commented:
Why does someone need this message? If ansible does not find the host key but the ssh client does, why does the user need to know? If the ssh client does not find the key as well, then ssh will ask the user for confirmation and notify the user about it as well. So there is not much use for normal users unless they want to debug something and then increasing the verbosity is reasonable IMHO.

I removed the whitespace changes, not sure if this really matters if the patch will not be accepted anyhow.

tyll commented:
Why does someone need this message? If ansible does not find the host key but the ssh client does, why does the user need to know? If the ssh client does not find the key as well, then ssh will ask the user for confirmation and notify the user about it as well. So there is not much use for normal users unless they want to debug something and then increasing the verbosity is reasonable IMHO.

I removed the whitespace changes, not sure if this really matters if the patch will not be accepted anyhow.
